tests/spread/integration/patch/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Integration tests for patch
+
+execute: |
+  rootfs="$(install-slices diffutils_bins patch_bins)"
+
+  chroot "${rootfs}/" patch --help
+  echo "Hello, World" > "${rootfs}/"testfile.txt
+  echo "This is a Patched Input" > "${rootfs}/"patchfile.txt
+  chroot "${rootfs}/" diff -u "${rootfs}/"testfile.txt "${rootfs}/"patchfile.txt \
+    > "${rootfs}/"tmp.patch
+  chroot "${rootfs}/" patch < "${rootfs}/"tmp.patch
```

Comment:
Can we validate that the contents of `testfile.txt` changed to `This is a Patched Input`?